miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -2,29 +2,62 @@ package com.rakuten.tech.mobile.miniapp.js
 
 import android.webkit.JavascriptInterface
 import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
 import com.rakuten.tech.mobile.miniapp.display.WebViewListener
 
+@Suppress("TooGenericExceptionCaught", "SwallowedException")
 /** Bridge interface for communicating with mini app. **/
 abstract class MiniAppMessageBridge {
     private lateinit var webViewListener: WebViewListener
 
     /** Get provided id of mini app for any purpose. **/
     abstract fun getUniqueId(): String
 
+    /** Post permission request from external. **/
+    abstract fun requestPermission(
+        callbackId: String,
```

Comment:
It seems like passing the `callbackId` makes things complicated for the host app. Could we instead pass a function which the Host App needs to invoke? Something like `callback: (isGranted: Boolean) -> Unit`?